README.md

```diff
@@ -30,6 +31,13 @@ Using ColPali removes the need for potentially complex and brittle layout recogn
 
 ![ColPali Architecture](assets/colpali_architecture.webp)
 
+## Supported models
+
+| Model                                        | Score on [ViDoRe](https://huggingface.co/spaces/vidore/vidore-leaderboard) 🏆 | License    | Comments |
+| -------------------------------------------- | ------------------------------------------------------------ | ---------- | -------- |
+| [vidore/colpali-v1.2](vidore/colpali-v1.2)   | 83.1                                                         | Gemma      |          |
+| [vidore/colqwen2-base](vidore/colqwen2-base) | 86.6                                                         | Apache 2.0 |          |
+
```

Comment:
@ManuelFay Can you help me complete this table today if you've got some time pls? There a few models missing + the comments column to fill.